generators/server/templates/build.gradle.ejs

```diff
@@ -145,6 +145,10 @@ task integrationTest(type: Test) {
     testLogging {
         events 'FAILED', 'SKIPPED'
     }
+    if (project.hasProperty('testcontainers')) {
```

Comment:
I think it doesn't make any difference, but I would say let's keep all properties together at the top of the build file.

Reply:
This works:
```
task integrationTest(type: Test) {
...
}
if (project.hasProperty('testcontainers')) {
    integrationTest {
        environment 'spring.profiles.active', 'testcontainers'
    }
}
```
But this one doesn't:
```
if (project.hasProperty('testcontainers')) {
    integrationTest {
        environment 'spring.profiles.active', 'testcontainers'
    }
}
task integrationTest(type: Test) {
...}
```

```
FAILURE: Build failed with an exception.

* Where:
Build file 'C:\Users\ecostanzi\dev\tmp\daigradle\build.gradle' line: 102

* What went wrong:
A problem occurred evaluating root project 'daigradle'.
> Could not find method integrationTest() for arguments [build_danqkclw8b0bg52d6bfrx5o9e$_run_closure6@69a2634e] on root project 'daigradle' of type org.gradle.api.Project.
```

I don't use gradle very often, so I'm open to suggestions :)

Reply:
Ah okay I see, was not sure about. Let's keep it as is.